devel-tools/build_test_library.cmake

```diff
@@ -85,6 +85,7 @@ execute_process(
   -D BUILD_SHARED_LIBS=${BUILD_SHARED_LIBS}
   -D WARNINGS_ARE_ERRORS=ON
   -D CMAKE_VERBOSE_MAKEFILE=ON
+  -D ENABLE_COVERAGE=ON
```

Comment:
I expected coverage reporting to be useful for manually working on the tests. What's its purpose inside the CI workflow, is there any automated treatment?

Reply:
I'm using that same CMake script also for manually checking the code on the local worktree.

An automated treatment would require additional tools, which I didn't set up yet. Right now, the library tests only parse the input, but do not run actual computations. So at the moment the flag is useful, just not yet at the stage where we would run a coverage analysis unattended. Do you feel that there is a reason *not* to include it at all?

Reply:
I see - my only concern was that there could be an impact on the CI run time. If you don't think that's the case, then I'm good.

Reply:
No, the code may run a bit slower, but it compiles in the same amount of time, and ccache can still skip that altogether if there is a hit (I just checked again).